app/helpers/profile_helper.rb

```diff
@@ -0,0 +1,19 @@
+#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+#   licensed under the Affero General Public License version 3 or later.  See
+#   the COPYRIGHT file.
+
+module ProfileHelper
+  def upper_limit_date_of_birth
+    minimum_year = AppConfig.settings.terms.minimum_age.get
+    if !(minimum_year)
+      minimum_year = 13
+    else
+      minimum_year = minimum_year.to_i
+    end
```

Comment:
Okay, just one last thing:

Let's prefer `unless` over `if !`:

```ruby
unless minimum_year
  minimum_year = 13
else
  minimum_year = minimum_year.to_i
end
```
Let's prefer `if` over `unless` when we can swap branches:


```ruby
if minimum_year
  minimum_year = minimum_year.to_i
else
  minimum_year = 13
end
```

Let's extract the assignment:

```ruby
 minimum_year = if minimum_year
  minimum_year.to_i
else
  13
end
```

And reduce it to a ternary:
```ruby
 minimum_year = minimum_year ?  minimum_year.to_i : 13
```